barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/goblin_recursive_verifier.cpp

```diff
@@ -28,9 +28,7 @@ void GoblinRecursiveVerifier::verify(const GoblinProof& proof)
         };
     translator_verifier.verify_translation(translation_evaluations);
 
-    // TODO(https://github.com/AztecProtocol/barretenberg/issues/1024): Perform recursive merge verification once it
-    // works with Ultra arithmetization
-    // MergeVerifier merge_verified{ builder };
-    // [[maybe_unused]] auto merge_pairing_points = merge_verifier.verify_proof(proof.merge_proof);
```

Comment:
I think it couldn't hurt to leave this pairing points stuff or a TODO or something just to highlight that this is incomplete. Maybe there's already an issue for the general lack of aggregation handling?